meteor/client/ui/FloatingInspectors/NoraFloatingInspector.tsx

```diff
@@ -186,7 +186,26 @@ export class NoraPreviewRenderer extends React.Component<{}, IStateHeader> {
 		this._observer.observe(e)
 	}
 
+	private _onNoraMessage = (msg: MessageEvent): void => {
+		if (msg.source !== this.iframeElement?.contentWindow) return
+		if (msg.data.event === 'nora-render') {
```

Comment:
I suspect that SonarLint will be complaining about not checking `msg.origin` on the `MessageEvent`. I think it would be good to add a check here to do

```typescript
const rendererUrl = new URL(this.state.noraContent?.previewRenderer)
if (rendererUrl.orign !== msg.origin) return
```